src/6.6/6.6.10.2.md

```diff
@@ -0,0 +1,33 @@
+---
+nav:
+  title: v6.6.10.2
+meta:
+  date: "2025-02-27"
+---
+
+# Release notes Shopware 6.6.10.2
+
+## Abstract
+
+This minor release contains two bug fixes.
+
+## System requirements
+
+* tested on PHP 8.2 and 8.3
+* tested on MySQL 8 and MariaDB 10.11
+
+## Fixed bugs
+
+* [PR7132:](https://github.com/shopware/shopware/pull/7132) With https://github.com/symfony/symfony/pull/59781/files, the definition *mailer.default_transport* was removed and set as a simply alias. Therefore it could not be accessed anymore in a compiler pass. But this should also be not necessary anymore.
+* [PR7121:](https://github.com/shopware/shopware/pull/7121) In languages, which use Apostrophes a lot (like french), the storefront broke because escaping was missing.
```

Comment:
```suggestion
* [PR-7121:](https://github.com/shopware/shopware/pull/7121) In languages, which use Apostrophes a lot (like French), the storefront broke because escaping was missing.
```